docs/dev/reference/dca/list.md

```diff
@@ -128,3 +128,53 @@ $GLOBALS['TL_DCA']['tl_example']['list']['operations'] = [
 | Key             | Value                             | Description                                                                                                        |
 |-----------------|-----------------------------------|--------------------------------------------------------------------------------------------------------------------|
 | route           | Symfony Route Name (`string`)     | The button will redirect to the given Symfony route.                                                               |
+
+{{% notice "note" %}}
+Since Contao **5.0** you do not have to define any settings for standard operations anymore. Instead you can give a list
+of which operations should be avilable for your data container. Contao will also check the appropriate
+[`contao_dc.<data-container>` permission](/framework/security/) for these operations.
+
+```php
+// contao/dca/tl_foobar.php
+$GLOBALS['TL_DCA']['tl_foobar']['config']['list']['operations'] = [
+    'edit',
+    'children',
+    'copy',
+    'cut',
+    'delete',
+    'toggle',
+    'show',
+    'toggle',
+];
+```
+{{% /notice %}}
+
+
+#### Toggle Operation
+
+{{< version-tag "5.0" >}} You can implement an automatic toggle operation for data containers that contain a boolean 
```

Comment:
That's 4.13 https://github.com/contao/contao/pull/3889

Reply:
What I documented only works in 5.0. In Contao 4.13 it's still more complicated and I would still recommend to use Haste there (see above).